plugins/GSdx/GSRendererDX11.cpp

```diff
@@ -154,7 +154,7 @@ void GSRendererDX11::EmulateTextureShuffleAndFbmask()
 		}
 
 	} else {
-		//m_ps_sel.fmt = GSLocalMemory::m_psm[m_context->FRAME.PSM].fmt;
+		m_ps_sel.dfmt = GSLocalMemory::m_psm[m_context->FRAME.PSM].fmt;
```

Comment:
``GSLocalMemory::m_psm[m_context->FRAME.PSM].fmt`` could return a potential 16 bit unsigned integer while dfmt is just 2 bits. would it be good to have some sort of checks or asserts here?